### ISSUE TITLE
Error while optimizing personal dataset

### ISSUE THREAD

**Issue by SouzaBB** [parklab/SigMA#59]
Hi. I'm following [this](https://github.com/parklab/SigMA/wiki/Optimize) tutorial to optimize my dataset for SigMA using colorectal cancer data with **confirmed MSI-H status**

I've created my input file using:
```
devtools::load_all()
genomes_matrix <- make_matrix("path_to_vcf_folder", file_type = 'vcf', ref_genome_name = 'hg19')
genomes <- conv_snv_matrix_to_df(genomes_matrix)
genome_file = 'colorectal.csv'
write.table(genomes, genome_file, sep = ',', row.names = F, col.names = T, quote = F)
```

After creating the input file, I've run:
```
input_file <- ("path_to_colorectal.csv")
tumor_type = 'crc'
remove_msi_pole <- T
```

Then, I'm trying to get `data_val`
`data_val <- find_data_setting(input_file, tumor_type, remove_msi_pole = remove_msi_pole)`

After executing this step, I'm getting the following **error** message:
```
Error in if (data1 == "msk_with_zeros") data1 <- "msk" :
    argument has zero length
Also: Warning messsages:
1: In min(mean_diff) :
  non missing argument to min; returning Inf
2: In min(median_diff) :
  non missing argument to min; returning Inf
```
Am I missing something?

Thanks.

**Comment by dgulhan-bio** [comment]
Hi, does it work with:
input_file <- ("colorectal.csv")

**Comment by SouzaBB** [comment]
@dgulhan-bio thanks for the response.

Unfortunately, using `input_file <- ("colorectal.csv")` returned the same error!

**Comment by dgulhan-bio** [comment]
I cannot reproduce the error if you don't mind sending the input file to me I can help more. The input file should have mutation counts in its first 96 columns where each row is a sample.

**Comment by SouzaBB** [comment]
[colorectal.csv](https://github.com/parklab/SigMA/files/6786774/colorectal.csv)

There goes the file... Just remembering that it was generated using the steps cited above. 

Thanks.

**Comment by dgulhan-bio** [comment]
Thanks I solved this issue there was a missing table entry in the average snv values for colorectal cancers. 

I noticed in the 4 samples you shared  there are germline mutation contaminations (see https://cancer.sanger.ac.uk/signatures/sbs/sbs54/). If this is a panel data without matched normals it would be better to filter the mutations using common variant databases gnomAD ExaC with a low maf cutoff. The 1000 mutations is too high for a panel data. Based on my experience for cases without matched normals 0.00001 (0.001%) MAF cutoff works the best. 

If you use more samples as input to decide the average total snvs (e.g. 50) that would also be better.

Hope this helps.
